compat/src/index.d.ts

```diff
@@ -67,7 +67,7 @@ declare namespace React {
 
 	export function unstable_batchedUpdates(callback: (arg?: any) => void, arg?: any): void;
 
-	export interface Children {
+	export const Children: {
```

Comment:
Updating the type to be a real object, not an interface for some other object to implement